int8 tracker_id
string req_type
---
int8 tracker_id
int32[] grid_ind
float64[] values